concordium-contracts-common/Cargo.toml

```diff
@@ -55,11 +55,19 @@ version = "1.0"
 path = "../concordium-contracts-common-derive"
 version = "1"
 
+[dependencies.anyhow]
+optional = true
+version = "1.0"
```

Comment:
I do not want to bring anyhow to contracts-common. Since the serialization is now part of a library we should define a precise error type for it and use thiserror to derive the relevant things for it.